data-prepper-logstash-configuration/src/main/java/com/amazon/dataprepper/logstash/model/LogstashAttribute.java

```diff
@@ -0,0 +1,22 @@
+package com.amazon.dataprepper.logstash.model;
+
+public class LogstashAttribute {
+    String attributeName;
```

Comment:
All of the fields are currently defined as "package protected" which means they can be accessed from other classes in the same package. They should all be private.